### PR TITLE
ASU-1358 | Get has_children value from customer if no application available

### PR DESCRIPTION
**Pull request by quyenlq**
Get has_children value from Customer if no application available, fix serializer so it'll return null value if no application attached to the reservation

### CODE REVIEW

File: application_form/api/sales/serializers.py [hunk 1]
```diff
@@ -46,22 +46,22 @@ class Meta:
 
 class SalesApartmentReservationSerializer(ApartmentReservationSerializerBase):
     applicants = ApplicantCompactSerializer(
-        source="application_apartment.application.applicants", many=True
+        source="application_apartment.application.applicants",
+        many=True,
+        allow_null=True,
```
Comment (tuomas777) [inline application_form/api/sales/serializers.py:51]: Hmm, why is `null` needed here? Shouldn't a non-existing value be an empty list instead of null?

Comment (quyenlq) [reply application_form/api/sales/serializers.py:51]: The `null` value is to handle the case that the application is `null`, i.e reservation created by sales person without application. Without `allow_null=True` that field will raise `AttributeError`. In any other case, it'll be an empty list if there is no applicant.

Comment (tuomas777) [reply application_form/api/sales/serializers.py:51]: Ok makes sense 👍

**Review by tuomas777 — APPROVED** [review]
LGTM 👍

**Pull request merged**